src/metabase/task/creator_sentiment_emails.clj

```diff
@@ -0,0 +1,118 @@
+(ns metabase.task.creator-sentiment-emails
+  (:require
+   [clojurewerkz.quartzite.jobs :as jobs]
+   [clojurewerkz.quartzite.schedule.cron :as cron]
+   [clojurewerkz.quartzite.triggers :as triggers]
+   [java-time.api :as t]
+   [metabase.analytics.snowplow :as snowplow]
+   [metabase.analytics.stats :as stats]
+   [metabase.config :as config]
+   [metabase.db :as mdb]
+   [metabase.driver.sql.query-processor :as sql.qp]
+   [metabase.email :as email]
+   [metabase.email.messages :as messages]
+   [metabase.models.setting :as setting]
+   [metabase.public-settings :as public-settings]
+   [metabase.public-settings.premium-features :as premium-features]
+   [metabase.task :as task]
+   [metabase.util.i18n :refer [deferred-tru]]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(setting/defsetting ^:private no-surveys
+  (deferred-tru "Enable or disable creator sentiment emails")
+  :type       :boolean
+  :default    false
+  :visibility :internal
+  :audit      :getter)
+
+(defn- fetch-creators
+  [has-whitelabelling?]
```

Comment:
"Creator" isn't a formal concept on the BE so it would be useful to have a docstring/comment here briefly explaining what defines a creator